tests/process_input_test.py

```diff
@@ -22,10 +22,10 @@
 def testProcessLink_whenGit_returnsCorrectType(
     github_link: str, gitlab_link: str, other_link: str
 ) -> None:
-    github_match = str(process_input.process_link(github_link))
-    gitlab_match = str(process_input.process_link(gitlab_link))
-    other_link = str(process_input.process_link(other_link))
+    github_match = process_input.get_link_type(github_link)
```

Comment:
This parametrization is weird. Pass the URL and expected type in the parameters.